### DISCUSSION TITLE
Can Gutenberg become Wix like editor?

### DISCUSSION THREAD

**Discussion by davidjeba** [WordPress/gutenberg#47106] [General]
Its really hard to use Gutenberg
Its has very limited flexibilities
Its nowhere compared to Elementor

Can Gutenberg differentiate itself to become a Wix like drag and drop editor?

**Comment by altafhpatel** [comment]
Well, i am using gutenberg from start and having the same issue using this, but then i creat a plugin ([custom block styles for gutenberg](https://wordpress.org/plugins/custom-block-styles/)) to overcome this problem. Below is the attached screenshots which created borders and other things which helps me to edit easily.
![Screenshot_1](https://user-images.githubusercontent.com/26499141/217366169-fec648ff-ce2d-4cd0-abec-578fc1a813a5.jpg)
